### ISSUE TITLE
css-layout should expose main.js + Layout.js 

### ISSUE THREAD

**Issue by kvey** [cljsjs/packages#95]
https://github.com/facebook/css-layout/blob/master/src/main.js

css-layout is written with mutability in mind - exposing the above in addition to Layout.js somehow would make the api exposed to clojurescript much saner. `computeLayout` returns nothing.

For general reference this snippet:
```clojure
(defn compute-layout [layout] 
  "css-layout is horrifically mutation based"
  (let [*nodes* (clj->js layout)]
    ((.-computeLayout js/computeLayout) ((.-fillNodes js/computeLayout) *nodes*))
    ((.-extractNodes js/computeLayout) *nodes*)))
```

Will work in the meantime (not entirely sure I'm adhering to the earmuffs convention properly there).

**Comment by martinklepsch** [comment]
ping @bobby @pandeiro

**Comment by bobby** [comment]
It appears that https://github.com/facebook/css-layout/blob/master/src/main.js pulls in Layout.js as an UMD module, which Closure compiler doesn't know about (https://github.com/umdjs/umd/issues/17).  We could probably pretty trivially re-create main.js in a Closure compiler-friendly way here in cljsjs, but we probably won't be able to use the upstream-provided main.js, unless I'm misunderstanding how UMD works.

**Comment by ThomasDeutsch** [comment]
can someone update this to v1.0?

**Comment by Deraen** [comment]
I'm going to presume this was fixed when css-layout was updated to version 1.0.

Please reopen or create a new issue if issues persist.
